.mergify.yml

```
@@ -149,3 +149,16 @@ pull_request_rules:
       comment:
         message: |
           This pull request has not been merged yet. Could you please review and merge it @{{ assignee | join(', @') }}? 🙏
+  - name: backport patches to 8.4 branch
+    conditions:
+      - merged
+      - label=backport-v8.4.0
```

@v1v can we change this automation to match our other backport labels?

```suggestion
      - label=backport-v8.4
```